src/browser.ts

```diff
@@ -11,7 +11,7 @@ export default function(history: History = createBrowserHistory()) {
   // there's only `view` which is the whole `pathname` from history perspective
 
   function syncToBrowser(s: BrowserState, push: boolean = true): void {
-    const pathname = `/${s.view}`;
+    const pathname = `/${s.view || ''}`;
```

Comment:
no I forgot this. It's responsibility of the app to enforce an always-there `view` key (and this is reflected in the static state type and, as a consequence, in `BrowserState` too). So no need for this default here.
I initially put the default there because on alinity pro we had:
```js
initialState = { /* ... no view key ... */ }
// and
transitionReducer = state => {
  if (state.view === 'main') { 
    state.view = defaultView
  }
  // ...
  return state;
}
```
I updated the check in `transitionReducer` to be `if (!state.view)` and it works 👍